src/components/Map.js

```diff
@@ -2,14 +2,32 @@ import React from 'react';
 
 import GoogleMapReact from 'google-map-react';
 
-const Map = () => {
-  const { center, zoom, key } = {
-    center: {
-      lat: 48.8566,
-      lng: 2.3522,
-    },
-    zoom: 11,
-    key: 'AIzaSyCbfV0IAdkkGv-9mmuAkUJNzCPPfGRO6v0',
+const Map = ({ data }) => {
+  const { centerCoords, zoom } = data;
+
+  const handleApiLoaded = (map, maps, data) => {
+    if (data && data.query.pages.length > 0) {
+      data.query.pages.forEach((location) => {
+        const { coordinates, pageid: id } = location;
+
+        const image = {
+          url:
+            location.thumbnail?.source ||
+            `${process.env.PUBLIC_URL}/placeholder.png`,
+          size: new maps.Size(24, 24),
+          origin: new maps.Point(0, 0),
+          anchor: new maps.Point(0, 24),
+        };
+
+        new maps.Marker({
```

Comment:
I'm impressed because:

1. The result is _not assigned_ to a variable. You've displayed markers but I don't understand how is it possible? 
2. Also, I have a little curiosity: did you consider using the `AnyReactComponent` component as the ["docs"](https://www.npmjs.com/package/google-map-react) to create the markers. Did you have some issues?

Reply:
Yep, initially, we used ``<AnyReactComponent/>```. But, according to the AC, pins need to drop onto the map once it's loaded. When working with ``<AnyReactComponent/>```, we couldn't find a way to animate markers

Reply:
Thank you for your detailed answers! I understood your reasons. Animations are very hard, congrats on it 😄!